public/scripts/app.js

```diff
@@ -47,66 +39,36 @@ $('ul.nav li').click(function(){
          `
        );
      }
-   }
-   else if(($(this).context.textContent)==="My Work")
-   {
-     //projects section
-
+    } else if(($(this).context.textContent)==="My Work") {
      $("#addProject").fadeIn();
      $("#addVacation").fadeOut();
-
-     console.log("Projects section");
      $.ajax({
        method:"GET",
-       url:'/api/profile/projects',
+       url:'/api/profile',
        success:function(data){
-         console.log(data);
-         $p_id=data[0]._id;
-         console.log($p_id);
-         $("#results").empty();
-
-         console.log(data[0].projects.length);
-         renderAllProjects(data);
-
```

Comment:
None of the above `console.log`s or variable definition are necessary. 

Moved the `$("#results").empty();` to the `renderAllProjects` function, as it is related to how you should render all the projects (empty the `#results` div first, then render in that div).